core/src/main/java/org/apache/brooklyn/util/core/internal/ssh/sshj/SshjTool.java

```diff
@@ -971,12 +987,12 @@ public Integer create() throws Exception {
                     errgobbler.start();
                 }
 
-                OutputStream output = shell.getOutputStream();
+                OutputStream shellStdin = shell.getOutputStream();
```

Comment:
this messes with the mind.  because _we_ write to stdin from our perspective it is an output stream, and the output from the shell we read from so it is an input stream.